build.gradle

```diff
@@ -423,16 +423,16 @@ task stopEC2(type: AWSLambdaInvokeTask) {
 //////////////////////////////////////////////////////////////////////
 task generateConfig {
     doLast {
-        if (!project.file('build/tscfg-0.8.1.jar').exists()) {
+        if (!project.file('build/tscfg-0.9.4.jar').exists()) {
```

Comment:
Replace `exists` with `isFile`. The current approach can lead to some problems in case there is no file but there is a directory with the same name.